docs/development_environment.md

```diff
@@ -49,7 +49,31 @@ sudo apt-get install python3-pip python3-dev python3-venv autoconf libssl-dev li
 
 ### Developing on Windows
 
-To develop on Windows, you will need to use the Linux subsystem. Follow the [WSL installation instructions](https://docs.microsoft.com/windows/wsl/install-win10) and install Ubuntu from the Windows Store. Once you're able to access Linux, follow the Linux instructions.
+To develop on Windows, you will need to use the Linux subsystem (WSL). Follow the [WSL installation instructions](https://docs.microsoft.com/windows/wsl/install-win10) and install Ubuntu from the Windows Store. Once you're able to access Linux, follow the Linux instructions.
+
+:::tip
+The first time a WSL distribution is started, and the default user account is created, the Windows drives will be mounted with all files owned by `root:root` instead of owned by the default user:
+```
+user@DESKTOP:/mnt/c/Users/user$ mount | grep mnt
+C:\ on /mnt/c type drvfs (rw,noatime,uid=0,gid=0,case=off)
+```
+
+This will cause the `setup` script to fail if the local repository is on a Windows drive. To recover, WSL must be restarted after which the Windows drived will be mounted with all files owned by the default user. This can be accomplished by simply restarting the computer, or by issuing the following command from a windows command prompt:
```

Comment:
```suggestion
This will cause the `setup` script to fail if the local repository is on a Windows drive. To recover, WSL must be restarted after which the Windows drives will be mounted with all files owned by the default user. This can be accomplished by simply restarting the computer, or by issuing the following command from a windows command prompt:
```